doc/nrf/security/crypto/crypto_supported_features.rst

```diff
@@ -31,848 +37,5673 @@ The following tables list hardware support for the PSA Crypto implementations in
 Cryptographic feature support
 *****************************
 
-The following sections list the supported cryptographic features and algorithms for each of the drivers: :ref:`nrf_cc3xx <crypto_drivers_cc3xx>`, :ref:`CRACEN <crypto_drivers_cracen>`, and :ref:`nrf_oberon <crypto_drivers_oberon>`.
+The following sections list the supported cryptographic features and algorithms for each of the Nordic Semiconductor device Series and cryptographicdrivers: :ref:`nrf_cc3xx <crypto_drivers_cc3xx>`, :ref:`CRACEN <crypto_drivers_cracen>`, and :ref:`nrf_oberon <crypto_drivers_oberon>`.
 The listed Kconfig options enable the features and algorithms for the drivers that support them.
 
 The Kconfig options follow the ``CONFIG_PSA_WANT_*`` + ``CONFIG_PSA_USE_*`` configuration scheme, which is described in detail on the :ref:`crypto_drivers` page.
 
 Key types and key management
 ============================
 
-The following table shows the Kconfig options for requesting Oberon PSA Crypto to use specific key types, as well as their support for each driver:
-
-.. list-table:: Key type configuration options and support per driver
-   :header-rows: 1
-   :widths: auto
-
-   * - Key type
-     - Configuration option
-     - :ref:`nrf_cc3xx <crypto_drivers_cc3xx>`
-     - :ref:`CRACEN <crypto_drivers_cracen>`
-     - :ref:`nrf_oberon <crypto_drivers_oberon>`
-   * - AES
-     - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_AES`
-     - Supported
-     - Supported
-     - Supported
-   * - Chacha20
-     - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_CHACHA20`
-     - Supported
-     - Supported
-     - Supported
-   * - ECC Key Pair
-     - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_ECC_KEY_PAIR`
-     - Supported
-     - Supported
-     - Supported
-   * - ECC Public Key
-     - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY`
-     - Supported
-     - Supported
-     - Supported
-   * - RSA Key Pair
-     - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR`
-     - Supported
-     - Supported
-     - Supported
-   * - RSA Public Key
-     - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY`
-     - Supported
-     - Supported
-     - Supported
+The following tables show the ``CONFIG_PSA_WANT_*`` Kconfig options for requesting Oberon PSA Crypto to use specific key types.
+The options are grouped by Series and driver, and support for each device is listed.
```

Comment:
Should it be made clearer that only the supported drivers for each series is shown in each table?